app/src/ui/autocompletion/autocompleting-text-input.tsx

```diff
@@ -132,6 +124,7 @@ export abstract class AutocompletingTextInput<
     const element = this.element!
     let coordinates = getCaretCoordinates(element, state.range.start)
     coordinates = {
+      ...coordinates,
```

Comment:
Why are we doing this now?

Reply:
There’s more keys on the `coordinates` object than `top` and `left`, so I’m copying them over to appease TS.